ci/dockerfiles/golang-builder/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:bionic
+
+# gcc for cgo
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		g++ \
+		gcc \
+		libc6-dev \
+		make \
+		pkg-config \
+                wget \
+                git \
+                ca-certificates \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_VERSION 1.11.5
+
+RUN set -eux; \
+	url="https://golang.org/dl/go${GOLANG_VERSION}.linux-amd64.tar.gz"; \
+	wget -O go.tgz "$url" 2>/dev/null; \
+	tar -C /usr/local -xzf go.tgz; \
+	rm go.tgz; \
```

Comment:
Hey,

These three lines can be done in a single one, e.g.:

```sh
curl -SL $url | tar xz -C /usr/local
```

(possible with `wget` too, I just don't recall the right flags :grin: )